plugins/dominant-color-images/hooks.php

```diff
@@ -107,7 +113,7 @@ function dominant_color_img_tag_add_dominant_color( $filtered_image, string $con
 	}
 
```

Comment:
This condition should be updated to use the tag processor, using `$processor->getAttribute('src')` and if it is not a string, return.